src/lib/shell/MainLoopSilabs.cpp

```diff
@@ -32,15 +32,19 @@ using chip::Shell::streamer_get;
 
 namespace {
 
-constexpr const char kShellPrompt[] = "matterCli > ";
+constexpr const char kShellPrompt[] = "matterCli> ";
 
 // max > 1
+bool haveCR = false;
```

Comment:
Why is this a global?

Reply:
This keeps track of Windows line feeds after ReadLine() has exitted and is ran again

Reply:
This keeps track of Windows line feeds after ReadLine() has exitted and is ran again